### PR TITLE
Reminder command updates v1.1

### PR DESCRIPTION
**Pull request by muserr**
Modified Reminder Command to sort reminders before displaying to user.
Included time based activities to be included in reminders.

### CODE REVIEW

File: src/main/java/duchess/logic/commands/ReminderCommand.java [hunk 1]
```diff
@@ -54,4 +55,16 @@ private void display(List<Task> reminderList, Ui ui) {
...
     }
+
+    /**
+     * Sorts deadlines in ascending order.
+     *
+     * @param reminderList list of reminders
+     * @return sorted list of reminders in ascending order
+     */
+    private List<Task> sortDeadlines(List<Task> reminderList) {
+        Collections.sort(reminderList, (Task thisTask, Task thatTask) -> thisTask.compareTo(thatTask));
```
Comment (mohideenik) [inline src/main/java/duchess/logic/commands/ReminderCommand.java]: The lambda is not necessary as `Task` is already comparable.

File: src/main/java/duchess/logic/commands/ReminderCommand.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
     @Override
     public void execute(TaskList taskList, Ui ui, Storage storage) throws DukeException {
         List<Task> reminderList = addDeadlines(taskList);
+        reminderList = sortDeadlines(reminderList);
```
Comment (mohideenik) [inline src/main/java/duchess/logic/commands/ReminderCommand.java]: The `sortDeadlines` function is redundant.

File: src/main/java/duchess/logic/commands/ReminderCommand.java [hunk 3]
```diff
@@ -28,6 +28,7 @@
     @Override
     public void execute(TaskList taskList, Ui ui, Storage storage) throws DukeException {
         List<Task> reminderList = addDeadlines(taskList);
```
Comment (mohideenik) [inline src/main/java/duchess/logic/commands/ReminderCommand.java]: The naming of `addDeadlines` is misleading.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -102,7 +102,7 @@ Example of usage:
 Expected outcome:
 
 ```
-You currently have these deadlines.
+You currently have these deadlines:
```
Comment (limsiying) [inline docs/UserGuide.adoc]: Oop sorry missed this out! The most up-to-date user guide that I sent just now has not been pr-ed. I'll add this change and resend to our wa group.

Comment (muserr) [reply docs/UserGuide.adoc]: Yepp thanks! It was just recently added.

**Pull request merged**